src/utils/db/queries.rs

```diff
@@ -283,3 +285,55 @@ fn input_table_name_for(aggregation_round: AggregationRound) -> &'static str {
         AggregationRound::Scheduler => "scheduler_witness_jobs_fri",
     }
 }
+
+pub async fn get_proof_time(
```

Comment:
I feel like this function has multiple functionalities, to either provide the proof time for a given batch (if given, the time period has no take here) or to provide a list of proof times within a period.

It'd be a better responsibility management to divide this into two different functions. The one responsible for retrieving a list of proof times within a period with the name suggested above (or a better one), and another one to get the proof time for a specific batch (given), could be `get_proof_time` or `get_proof_time_for_batch`.